### DISCUSSION TITLE
Open up for group discussion: Disease Progression

### DISCUSSION THREAD

**Discussion by rtmill** [OHDSI/OncologyWG#475] [Ideas]
See overall description of task here: https://github.com/OHDSI/OncologyWG/issues/469

**Comment by rtmill** [comment]
**Please provide any feedback and, if able, verify that one or more of these concept sets would be sufficient for your data.** 


# Problem Space

Create discrete set of standard concepts to encapsulate disease "status" observations over time. Likely the majority of these observations will come from notes and consequentially be curated or NLP derived. 

One difficulty here will be when the determination of trajectory are not clear, specifically when there are contradictory observations, e.g. one tumor improved at the same time as another worsened. 

# Proposed Solutions

Some suggested concept sets: 
1) DFCI (custom note extraction value set) 

ID | Value
-- | --- 
1	|Improving/Responding
2	|Stable/No change
3	|Mixed
4	|Progressing/Worsening/Enlarging
5	|Not stated/Indeterminate

2) LOINC [97509-4](https://athena.ohdsi.org/search-terms/terms/1617595) "Cancer disease progression" 

CONCEPT_NAME|CONCEPT_CODE|VOCABULARY
-- | -- | --
Improved | 45883920 | LOINC
No edvidence of disease | 1620321 | LOINC
Stable | 45879403 | LOINC
Undetermined | 45880649 | LOINC
 Worsened | 1620411 | LOINC

3) mCode 'Condition Status Trend Value Set'
http://hl7.org/fhir/us/mcode/ValueSet-mcode-condition-status-trend-vs.html

Code | System | Display
-- | -- | --
268910001	|http://snomed.info/sct|	Patient's condition improved (finding)
359746009	|http://snomed.info/sct|	Patient's condition stable (finding)
271299001	|http://snomed.info/sct|	Patient's condition worsened (finding)
709137006	|http://snomed.info/sct|	Patient condition undetermined (finding)
USCRS-352236	|http://hl7.org/fhir/us/mcode/CodeSystem/snomed-requested-cs|	Cancer in complete remission(finding)
USCRS-352237	|http://hl7.org/fhir/us/mcode/CodeSystem/snomed-requested-cs|	Cancer in partial remission (finding)

**Comment by cgreich** [comment]
@rtmill:

Wait. We have the dynamic Episodes for that purpose. That's exactly their point. Why do we need another parallel set?

**Reply by rtmill** [reply]
This entire sprint is to focus on conventions for representing data at the 'low level', outside of how they may or may not be used in episodes. 

For this case, it would be an observation or a measurement with a FK back to the condition record to enable a standard recording of progression over time.

**Reply by jam560** [reply]
Agreeing with Robert: My understanding is that an observation of NED (based on an imaging report in DFCI's curation process), for example, would be interpreted by the episode extraction process to signify the end of a disease episode. The other Disease Progression statuses would signify other semantics of episodes.

Given the values listed, how would episode extraction detect start of a recurrence episode? Would we expect to see an NED observation and then at a later time a "worsened" observation? There isn't a Disease Progression status for representing "initial" presence of cancer in the first scan, or first non-NED scan after NED.

**Comment by rtmill** [comment]
If we are going to do any meaningful longitudinal analysis of the disease throughout the patient journey it seems records of disease status whenever available is crucial. 

Outside of tumor registries (which of those I'm aware of have a single "update" record which may contain this sort of data), this will most likely be coming from NLP or manually curated data. Consequently, all we need to do is define a discrete list that is applicable to the wider community and ensure those concepts exist and are standard. 

Another value was mentioned for a study that could potentially be added to the set was "pseudo progression" though I'm not sure of the specific context or analytic value.

**Comment by gkennos** [comment]
Commented on the task, but realised it should probably go here first - vote +1 for LOINC coding as preferred option, and also adding support for needing observations as well as episodes

**Comment by cgreich** [comment]
@jam560:

> how would episode extraction detect start of a recurrence episode?

Good question. Not easy. Which is why we had the idea that if somebody does the job for us, we will just use the data and put them in as "low level". However, my question is why do we need a set of concepts that are similar but differently named. The list we have is fine unless proven otherwise.

If we don't have that, we are looking at auto-abstraction. That will be ugly, but it might be infinitely better than not having it. The logic is the following:

We have diagnostic events and treatment events. The initial diagnosis sets the train off as "progressive disease", since the cancer grew to its current state and was detected. The assumption is that it will continue until there is a treatment event. If the treatment is surgery we expect an immediate "response". If the treatment is chemo or radio we need to wait till there is a diagnostic event telling us there is response. If not, it is still "progressive". Once it is in "response", we wait for either another diagnostic event telling us it is now "progressive" (or maybe stable), or a new treatment event indicating that the disease is again "progressive" (could be the patient has new symptoms or the diagnostics were done in another center). And so on.

We know that such an algorithm is wicked, but would be a really cool research project.

Makes sense?

**Comment by askanter** [comment]
This is an interesting discussion of terminology (ignoring Christian's initial comment about where this should be captured). My first reaction is that the LOINC code for progression would actually apply to any disease state and so storing it in a cancer-specific concept makes it less generalizable. However, the mCODE value set includes cancer-specific states, but these also overlap with diagnosis criteria such as leukemia "in remission". That potentially sets you up for data quality problems or at least disagreements in the data. If you are trying to ascertain progression there are multiple ways that could be understood. I like a more generic set of criteria as it would be more likely to be accurate and would apply across numerous use cases. Specific criteria such as "in remission" could be captured as separate data elements or one that specifically targets presence of tumor cells (versus overall status of patient's condition). What about splitting them?

**Comment by cgreich** [comment]
@askanter: Not sure what you are asking. Do you want another set of dynamics of the disease? Or you don't want to reserve the current ones for cancer (they are not, btw)?

**Reply by askanter** [reply]
I think my recommendation was to have a separate, generalizable, disease-progression concept and value set that did not contain oncology specific codes. IF you chose a separate oncology-specific concept which was narrowly defined based on tumor status, which would not try to capture overarching disease state, but that included remission or "no malignant cells", you would need to keep these in synch with the diagnosis concepts. This was similar to my concern about metastatic disease which could could be captured at the condition level and the modifier level.

**Comment by cgreich** [comment]
I see. Right now we have:

- Partial Remission
- Progression
- Remission
- Stable Disease

Does that not work?





































)

**Comment by rtmill** [comment]
Per the EU workshop discussion and @gkennos previous comments, seems RECIST is highly recommended as the way to go:
```

4.3.1. Evaluation of target lesions
Complete Response (CR): Disappearance of all target lesions.
Any pathological lymph nodes (whether target or
non-target) must have reduction in short axis to
<10 mm.
Partial Response (PR): At least a 30% decrease in the sum of
diameters of target lesions, taking as reference the
baseline sum diameters.
232 EUROPEAN JOURNAL OF CANCER 45 (2009) 228 – 247
Progressive Disease (PD): At least a 20% increase in the sum
of diameters of target lesions, taking as reference
the smallest sum on study (this includes the baseline
sum if that is the smallest on study). In addition to
the relative increase of 20%, the sum must also demonstrate an absolute increase of at least 5 mm. (Note:
the appearance of one or more new lesions is also
considered progression).
Stable Disease (SD): Neither sufficient shrinkage to qualify for
PR nor sufficient increase to qualify for PD, taking as
reference the smallest sum diameters while on study
```

**Reply by jam560** [reply]
In our ETL project we are now mapping the manually curated data we have for a few thousand patients and I need to decide how to (at least temporarily) represent this (in Oncology 1.0). 

When we built our curation project at DFCI we intentionally did not use RECIST because it requires radiologists to assess. Real world imaging reports do not contain sufficient information to inform RECIST's precise size thresholds. We use only the "impressions" section of imaging reports and answer two questions: 1) is there presence of cancer? 2) If yes, worse, stable, improved?  From that we can derive the 5 or 6 concepts we are discussing here. We have shown we can auto-curate those questions very accurately.

Athena contains one more concept than @cgreich's list: Complete Remission

https://athena.ohdsi.org/search-terms/terms?conceptClass=Disease+Dynamic

And I'd argue we need the "mixed" concept. Neither "partial remission" or "progression" cover the case of an imaging report that says some lesions improved and some worsened.
